src/dataVersion.ts

```diff
@@ -0,0 +1,43 @@
+/*
+This file is responsible for switching views in original database like `ojp`, `routeing`
+etc. from for example `ojp_2019_02_02` to `ojp_2019_02_03`
+So in case of data failure we can simply switch views between old still working databases
+ */
+import {Container} from "./cli/Container";
+import {viewsSqlFactory} from "../config";
+import {ojpViews} from "../config/gtfs/views";
+import {OfflineDataProcessor} from "./database/OfflineDataProcessor";
+
+const container = new Container();
+
+const f = async () => {
+
+  const databaseWithViews = process.argv[2];
+  const databaseWithData = process.argv[3];
+
+  if (databaseWithViews.length === 0 ||
```

Comment:
I would check if those variables are not null first. If you execute script from cmd without args, then `process.argv[2]` will be null, and then `.length` called on null will shout "Cannot read property 'length' of null"